content/achievements.md

```diff
@@ -19,6 +19,17 @@ These ones were quite successful:
     <h2 class="timelineMajorMarker"><span>2015</span></h2>
     <dl class="timelineMinor">
 
+      <dt id="event25"><a>Robotic Battle Kosice</a></dt>
+      <dd class="timelineEvent" id="event25EX" style="display: none; ">
+        <div><ul>
+             <li>Racing-Electronics-Knock-out - 1st prize</li>
+             <li>Racing-Electronics-Race - 2nd prize</li>
```

Comment:
@Adman just so that we stay consistent with the other years can you add a bit more contextual information here? Something like

```
Racing-Electronics-Knock-out - 1st prize (previously called Racing)
```

or something else, along those lines.